app/controllers/reports/regions_controller.rb

```diff
@@ -26,12 +26,18 @@ def show
     @new_registrations = @last_registration_value - (@data[:cumulative_registrations].values[-2] || 0)
     @adjusted_registration_date = @data[:adjusted_registrations].keys[-4]
 
-    if @region.is_a?(FacilityGroup) || @region.is_a?(FacilityDistrict)
+    if @region.district_region?
```

Comment:
There's [one more touchpoint](https://github.com/simpledotorg/simple-server/pull/1477/files#diff-15a670f34e2a19569d76af44251cfd75268f11cfd403b931241305f2c5becfd9R75) now that needs this `district_region?` applied.

Reply:
Hm, this link isn't working - can you point me to the file / line number?

Reply:
Funtimes 🤦  Here ya go

https://github.com/simpledotorg/simple-server/blob/master/app/controllers/reports/regions_controller.rb#L69

It's the `download` action in the same controller.